packages/store-api/src/typeDefs/aggregateOffer.graphql

```diff
@@ -0,0 +1,10 @@
+type StoreAggregateOffer {
+  # Highest spot price amongst all sellers
+  highPrice: Float!
```

Comment:
Why should we make all fields required? Is this to compensate for the fact that we generate TypeScript types from the GraphQL schema?

Reply:
Not only to compensate the fact we are generating the typescript typings, but also to the fact that we need this data to render on the screen/metadata

Reply:
But by doing this we're only guaranteeing that resolvers for each of those fields are not able to return `null`, and also making it required for all resolvers to always contain an implementation. 

We're not preventing users from querying only a subset of these fields, right? They don't always need all of the data to render in their stores.

Reply:
I'm not sure if I got your point, but yes, If the person doesn't need that field it won't query it on the frontend, but if it needs, it must be required on the backend